<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 238, 218);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>335</width>
      <height>430</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>335</width>
      <height>430</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>335</width>
      <height>430</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_year">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;year:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_year">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_month">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;month:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_month">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EnterButton">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(135, 206, 250, 255), stop:1 rgba(240, 248, 255, 255));</string>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>530</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>125</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 209, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>68</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>68</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>68</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label_time</zorder>
   <zorder>textBrowser</zorder>
   <zorder>SettingButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_A">
    <property name="title">
     <string>Add(&amp;A)</string>
    </property>
    <addaction name="AddDateFestival"/>
    <addaction name="separator"/>
    <addaction name="AddWeekDayFestival"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDelet_D">
    <property name="title">
     <string>Delet(&amp;D)</string>
    </property>
    <addaction name="actionDeletFestival"/>
   </widget>
   <widget class="QMenu" name="menuFind_F">
    <property name="title">
     <string>Find(&amp;F)</string>
    </property>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuSearch_S">
    <property name="title">
     <string>Sort(&amp;S)</string>
    </property>
    <addaction name="actionSearch_and_Sort"/>
   </widget>
   <widget class="QMenu" name="menuReset_R">
    <property name="title">
     <string>Reset(&amp;R)</string>
    </property>
    <addaction name="ResetDateFestival"/>
    <addaction name="separator"/>
    <addaction name="ResetWeekDayFestival"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionInstructions"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAdd_A"/>
   <addaction name="menuDelet_D"/>
   <addaction name="menuFind_F"/>
   <addaction name="menuSearch_S"/>
   <addaction name="menuReset_R"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="AddDateFestival">
   <property name="text">
    <string>DateFestival</string>
   </property>
  </action>
  <action name="AddWeekDayFestival">
   <property name="text">
    <string>WeekDEayFestival</string>
   </property>
  </action>
  <action name="ResetDateFestival">
   <property name="text">
    <string>DateFestival</string>
   </property>
  </action>
  <action name="ResetWeekDayFestival">
   <property name="text">
    <string>WeekDayFestival</string>
   </property>
  </action>
  <action name="actionDeletFestival">
   <property name="text">
    <string>DeletFestival</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionSearch_and_Sort">
   <property name="text">
    <string>Search and Sort</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
